news/2022-06-30-community-contributors-2021.md

```diff
@@ -0,0 +1,101 @@
+
+As if this were not enough on its own, he is never far from hand in any macOS-related threads, ever ready and willing to help those who need it. His efforts have easily influenced thousands at this point, and we are enormously grateful for them.
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/huu.jpg)](https://osu.ppy.sh/users/6044237)
+
+Entrusted with the wardenship of Project Loved by its previous maintainers, huu was saddled with no small task, and has simply excelled admirably in his time heading the project. From organizing mappers, writers, and captains alike, huu has delved right into the thick of things and helped ensure that the legacy of Project Loved was set to continue in the present day, and it is a sure thing to say that it would not be anywhere near as successful as it presently is without his dedication and attention.
+
+Though he has since left the project to move on to other interests elsewhere, we are deeply grateful for his contributions, and are proud to welcome him to the list.
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/olibomby.jpg)](https://osu.ppy.sh/users/6573093)
+
+The mastermind behind the deceptively simple-sounding toolkit known only as Mapping Tools, OliBomby's developmental efforts have trivialized previously impossibly arduous tasks of creative complexity in the mapping scene, making things that were simply not possible before into reality. With nearly two dozen tools crammed into one package, the lifetime hours-saving of his package must number well into the thousands at this point.
+
+It should come as no surprise that when the modding and mapping community was asked to submit nominations for our list, OliBomby received the highest number of any individual we have ever seen thus far... several times over.
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/frukoyurdakul.jpg)](https://osu.ppy.sh/users/7612550)
+
+Continuing a long-standing trend of developers striving to make the world better one quality-of-life utility at a time, frukoyurdakul has earned the eternal love of the osu! taiko community with his timidly-named utility titled "Manage Beatmap", more commonly referred to as the "SV tool" or the "fruko tool" for good reason. 
+
+It is said that this singular utility is responsible for the birth of an entirely new mapping style, an accolade highly supported by the absolute deluge of nominations he received from the mapping and modding community.
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/leoflt.jpg)](https://osu.ppy.sh/users/3668779)
+
+A constant presence in the tournament scene for many years and recently brought aboard as part of the official osu! World Cup management crew, LeoFLT's tireless efforts largely speak for themselves. From handling programming and statistics-related tasks, to spreadsheets, website designs and much more, his proven technical prowess is only the tip of the iceberg.
```

Comment:
```suggestion
A constant presence in the tournament scene for many years and recently brought aboard as part of the official osu! World Cup management crew, LeoFLT's tireless efforts largely speak for themselves. From handling programming- and statistics-related tasks, to spreadsheets, website designs and much more, his proven technical prowess is only the tip of the iceberg.
```

Reply:
note that [this suggestion](https://github.com/ppy/osu-wiki/pull/7577#discussion_r910866463) has the same change

Reply:
unless i'm blind there's no terminating dash there nor does it join to another word, is the suggestion missing something?